### DISCUSSION TITLE
import Stock transactions

### DISCUSSION THREAD

**Discussion by pranab-bala** [moneymanagerex/moneymanagerex#6084] [General]
is there a possibility for importing stock transactions ?

**Comment by tactilis** [comment]
> is there a possibility for importing stock transactions ?

Currently, this is not possible.

Where do the stock transactions come from?

What format do they have?
Could you post an example file here please?  Note that you may need to .zip the file first if github doesn't allow the file type to be uploaded.

**Reply by pranab-bala** [reply]
Hi Martin,
Thanks for quick response. Please find sample csv data and also in qif format.
 I was exploring this application and looks very useful. except that it does not allow import of security transactions. I think it is very important feature to be in the next level. Practically, users would like to export data from other application and import it weekly/monthly as minimum. otherwise, it would be a data entry Job

best regards
[sampledata.zip](https://github.com/moneymanagerex/moneymanagerex/files/11994371/sampledata.zip)

**Comment by tactilis** [comment]
Issue https://github.com/moneymanagerex/moneymanagerex/issues/6085 has been raised for this feature request.

**Please continue any discussion of the feature there.**
